src/gluonts/dataset/split/splitter.py

```diff
@@ -266,7 +276,9 @@ class OffsetSplitter(pydantic.BaseModel, AbstractBaseSplitter):
     A positive offset indicates how many observations since the start of each
     series should be in the training slice; a negative offset indicates how
     many observations before the end of each series should be excluded from
-    the training slice.
+    the training slice. Please make sure that the number of excluded values
+    is enough for the test case, i.e., at least ``prediction_length`` (for
+    ``rolling_split`` multiple of ``prediction_length``) values are left off.
```

Comment:
Minor: why is this in the class docstring for `OffsetSplitter` but in the argument description for `DateSplitter`? :-)

Reply:
I was asking myself too :D Rolling_split was explained in the main docstring instead of in the parameters section. I assume whoever did that had a reason...
I'll put it into the parameters section now.